Modules/AppointmentSlots/Resources/views/select_appointments/interview_schedule_error_message.blade.php

```diff
@@ -0,0 +1,19 @@
+@extends('appointmentslots::layouts.master')
+
+@section('content')
+
+<div>
+    <div class="placing-of-content text-center">
+        <div class="mx-auto exclamation-icon h-44 w-44">{!! file_get_contents(public_path('icons/exclamation-circle.svg')) !!}</div>
+    </div>
+    <h1 class="placing-of-content text-center">Page Not found</h1><br>
+
+    <p class="error-message text-center fz-20 leading-25">
+        {{__('Looks like your interview is already scheduled. Please check your inbox for confirmation.') }}<br>

```

Comment:
Correct at other places as well
```suggestion
        {{ __('Looks like your interview is already scheduled. Please check your inbox for confirmation.') }}<br>

```